### DISCUSSION TITLE
can't claim benefits

### DISCUSSION THREAD

**Discussion by rizxy-c** [community/community#61534] [GitHub Education]
### Hi everyone,

Sure, here's the translation into English:

"I would like to inquire about this matter. Yesterday, I applied for the GitHub Student Developer Pack, and after some time, my application was approved. However, I haven't received any benefits yet; there's only a page with the following message:

"Hi, my-name You were last verified as a student on July 21, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."

It has been 3 days since it was approved, but I still cannot claim the benefits
What should I do to overcome this situation?

![Screenshot_20230722-054551152 (1)](https://github.com/community/community/assets/104349128/975df023-b87a-48ee-b344-18a0c76facce)

**Comment by yetkinkrsngr** [comment]
me too still waiting :S
